### PR TITLE
Meshctl check

### PR DESCRIPTION
**Pull request by harveyxia**
BOT NOTES: 
resolves https://github.com/solo-io/gloo-mesh/issues/1720

### CODE REVIEW

File: pkg/meshctl/commands/check/check.go [hunk 1]
```diff
@@ -15,9 +15,9 @@ func Command(ctx context.Context, inCluster bool) *cobra.Command {
 	}
 
 	cmd.AddCommand(
+		allCmd(ctx, inCluster),
```
Comment (EItanya) [inline pkg/meshctl/commands/check/check.go:18]: Can check also call "all" by default

File: changelog/v1.1.0-beta33/meshctl-check-all.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+changelog:
+  - type: NEW_FEATURE
+    issueLink: https://github.com/solo-io/gloo-mesh/issues/1720
+    description: All `meshctl check all` as a replacement for pre-1.1 `meshctl check`. Runs post install checks for management cluster and all remote clusters, using the meshctl config file as the source of truth for all Gloo Mesh clusters.
```
Comment (ashleywang1) [inline changelog/v1.1.0-beta33/meshctl-check-all.yaml:4]: Is there any reason we can't use `meshctl check` for the functionality of `meshctl check all`? Some of our own scripts even use it, and this change might break them. (See our bugreport command https://github.com/solo-io/gloo-mesh/blob/main/pkg/meshctl/commands/debug/report/internal/bugreport/bugreport.go#L268)

Comment (harveyxia) [reply changelog/v1.1.0-beta33/meshctl-check-all.yaml:4]: I mistakenly thought that any flags for a command with subcommands must also be included in the subcommands. But `cmd.Flags` does not persist to subcommands. Made the change in the latest commit

File: docs/content/reference/cli/meshctl_check_all.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
```
Comment (EItanya) [inline docs/content/reference/cli/meshctl_check_all.md:1]: This file should be deleted

**Review by EItanya — COMMENTED** [review]
LGTM, just needs codegen

**Review by EItanya — APPROVED** [review]

**Review by EItanya — APPROVED** [review]

**Review by EItanya — APPROVED** [review]

**Pull request merged**